MetadataExtractor/Util/DateUtil.cs

```diff
@@ -18,7 +18,7 @@ public static bool IsValidTime(int hours, int minutes, int seconds)
                minutes is >= 0 and < 60
                && seconds is >= 0 and < 60;
 
-#if NET8_0 || NETSTANDARD2_1
+#if NET8_0_OR_GREATER || NETSTANDARD2_1
```

Comment:
Thanks for fixing this. I searched and couldn't find any other instances.

Reply:
Yep, also searched -- and only one I found!